zoo.py

```diff
@@ -0,0 +1,30 @@
+
+class Animal:
+    def __init__(self, name):
+        self.name = name
+        self.age = 2
+
+    def speak(self):
+        print(self.age)
+        print("I am an animal")
+
+
+class Dog(Animal):
+    def speak(self):
+        print("I am a dog")
+
+
+class Bear(Animal):
+    def speak(self):
+        print("I am a bear named", self.name)
+
+
+animal = Animal(name='fluffy')
```

Comment:
All names (here and on the lines below) should start with capital letters.

Reply:
made the change